client/src/components/admin/admin.js

```diff
@@ -38,13 +38,13 @@ class Admin extends Component {
         if (id) {
             this.setState({ selectedID: id });
         } else {
-            this.setState({ selectedID: null });
+            this.setState({ selectedID: null, newItemBeingCreated: false });
         }
     }
 
     // Sets selectedID to "newItem", making a menu item editor open to create a new menu item
     openCreateItem() {
-        this.setState({ selectedID: "newItem" });
+        this.setState({ newItemBeingCreated: "true" });
```

Comment:
```suggestion
        this.setState({ newItemBeingCreated: true });
```
Not supposed to be a string. Someone might make a check which requires a boolean with the value true later and then this could cause that to fail